perfscale_regression_ci/scripts/scalability/descheduler/common_func.sh

```diff
@@ -0,0 +1,50 @@
+
+function wait_for_pod_creation() {
+  COUNTER=0
+  creating=$(oc get pods -A | grep $1 | egrep -c -e "Pending|Creating|Error" )
```

Comment:
Lets add another variable to take in the object_type like in this function: https://github.com/openshift/svt/blob/6a0babd4e93c1107b4ac6c5de804a83bf0a5f481/perfscale_regression_ci/scripts/common.sh#L25